sky/tools/create_full_ios_framework.py

```diff
@@ -168,9 +170,15 @@ def zip_archive(dst):
       'zip', '-r', 'artifacts.zip', 'gen_snapshot_arm64', 'Flutter.xcframework'
   ],
                         cwd=dst)
+  if (os.path.exists(os.path.join(dst, 'Flutter.dSYM', 'Contents'))):
```

Comment:
Do the scripts already know not to generate external symbols except in release mode?

Reply:
Not sure, but for engine_v2 recipes we are calling explicit ninja targets rather than building the full configuration. This is an example: https://github.com/flutter/engine/pull/35366/files